hera_sim/simulate.py

```diff
@@ -321,12 +326,24 @@ def _check_compatibility(self):
         if self.data.phase_type != "drift":
             raise CompatibilityException("The phase_type of the data must be 'drift'.")
 
-    def _iterate_antpair_pols(self):
+    def _iterate_antpair_pols(self, **kwargs):
```

Comment:
prefer using explicit arguments here. You can make sure it's passed as a keyword by using `_iterate_antpair_pols(self, *, use_baselines=None)`